File: Chapter 02/C Programs/2_11.c
#include<stdio.h>
void main()
{
int a,b,c;
printf("Enter 3 sides of a triangle: ");
scanf("%d%d%d",&a,&b,&c);
if (a==b && b==c)
printf("It is an equilateral triangle: ");
else if (a==b || b==c || c==a)
printf("It is an isosceles triangle");
else
printf("It is a scalene triangle");
}

File: Chapter 02/Python programs/2_11.py
a=int(input("Enter side 1 of a triangle "))
b=int(input("Enter side 1 of a triangle "))
c=int(input("Enter side 1 of a triangle "))
if a==b and b==c:
    print("It is an equilateral triangle")
elif a==b or b==c or c==a :
    print("It is an isosceles triangle")
else:
    print("It is a scalene triangle")
